src/transport/raw/MessageHeader.cpp

```diff
@@ -207,6 +208,17 @@ CHIP_ERROR PayloadHeader::Decode(Header::Flags flags, const uint8_t * const data
         mVendorId.ClearValue();
     }
 
+    mExchangeFlags.SetRaw(header);
```

Comment:
Whad does the setInitiator do?

I believe the initiator flags should be based on mExchangeFlags, so checking mExchangeFlags to set mExchangeFlags seems like a noop.

Reply:
Yes, mExchangeFlags.SetRaw(header); has already handled flag set, the setInitiator operation is not needed. Acked.